UnitTests/SequenceExtensionsFixture.cs

```diff
@@ -21,6 +22,29 @@ public void PerformSequence()
 		}
 
 		[Fact]
+		public void PerformSequenceAsync()
+		{
+			var mock = new Mock<IFoo>();
+
+			mock.SetupSequence(x => x.DoAsync())
+				.ReturnsAsync(2)
+				.ReturnsAsync(3)
+				.ThrowsAsync(new InvalidOperationException());
+
+			Assert.Equal(2, mock.Object.DoAsync().Result);
```

Comment:
I'm for it. The minimal officially supported  version now days is 4.5.2. So I'd jump to 4.6.1.